.github/workflows/e2e_courses_and_topics.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: e2e courses and topics
+
+on:
+  push:
+    branches-ignore:
+      - dependabot/**
+
+jobs:
+  prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch_name;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+  tag-courses_and_topics:
+    uses: hpi-schul-cloud/end-to-end-tests/.github/workflows/e2e.yml@${{ steps.extract_branch.outputs.branch_name }}
```

Comment:
This won't work the work flow need at starts the at this point the branch name :(

At the one of my runs on freiday at the server repo i tried an other way for this issus and this way also don't work.

https://github.com/hpi-schul-cloud/schulcloud-server/actions/runs/1319960984/workflow#L67